libr/core/cmd_search.c

```diff
@@ -1235,6 +1235,14 @@ static int r_core_search_rop(RCore *core, RInterval search_itv, int opt, const c
 	RIOMap *map;
 	RAsmOp asmop;
 
+	Sdb *gadgetSdb;
+	if (!(gadgetSdb = sdb_ns (core->sdb, "gadget_sdb", false))) {
+		gadgetSdb = sdb_ns (core->sdb, "gadget_sdb", true);
```

Comment:
just call it once with true, will have the same effect and avoid the if and null assignments

Reply:
I don't think it will have the same effect. What I want is to create the ns if it doesn't exist, but have gadgetSdb set as NULL pointer if the ns already exist, because I don't want to touch it again (the search is stored already).